Index: astro_2.0/features/Bounties/components/BountiesListView/components/AmountContent/AmountContent.module.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/components/BountiesListView/components/AmountContent/AmountContent.module.scss b/astro_2.0/features/Bounties/components/BountiesListView/components/AmountContent/AmountContent.module.scss
--- a/astro_2.0/features/Bounties/components/BountiesListView/components/AmountContent/AmountContent.module.scss	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/astro_2.0/features/Bounties/components/BountiesListView/components/AmountContent/AmountContent.module.scss	(date 1643889193673)
@@ -9,7 +9,11 @@
 .item {
   align-items: center;
   display: flex;
-  padding: 20px 0;
+  padding: 0;
+
+  @include tablet-landscape {
+    padding: 20px 0;
+  }
 
   &:first-of-type {
     width: 130px;
Index: astro_2.0/features/pages/nestedDaoPagesContent/BountiesPageContent/BountiesPageContent.module.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/pages/nestedDaoPagesContent/BountiesPageContent/BountiesPageContent.module.scss b/astro_2.0/features/pages/nestedDaoPagesContent/BountiesPageContent/BountiesPageContent.module.scss
--- a/astro_2.0/features/pages/nestedDaoPagesContent/BountiesPageContent/BountiesPageContent.module.scss	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/astro_2.0/features/pages/nestedDaoPagesContent/BountiesPageContent/BountiesPageContent.module.scss	(date 1643903420065)
@@ -3,12 +3,17 @@
 
 .root {
   display: grid;
-  grid-row-gap: 24px;
+  grid-row-gap: 0;
   grid-template-areas:
     'header'
     'content';
   grid-template-columns: 1fr;
-  padding-top: 40px;
+  padding-top: 24px;
+
+  @include tablet-landscape {
+    grid-row-gap: 24px;
+    padding-top: 40px;
+  }
 
   @include desktop {
     padding-top: 56px;
@@ -30,9 +35,23 @@
 }
 
 .newProposalButton {
+  bottom: 80px;
   font-size: 14px;
-  margin-left: 24px;
+  left: 24px;
+  position: fixed;
+  right: 24px;
   text-transform: capitalize;
+  width: calc(100% - 48px);
+  z-index: 10;
+
+  @include tablet-landscape {
+    bottom: unset;
+    left: unset;
+    margin-left: 24px;
+    position: relative;
+    right: unset;
+    width: auto;
+  }
 }
 
 .filters {
@@ -44,6 +63,7 @@
   margin-left: auto;
 
   .filter {
+    align-items: center;
     margin-right: 12px;
   }
 
@@ -51,3 +71,16 @@
     margin-right: 6px;
   }
 }
+
+.filterMenu {
+  right: 0;
+  top: 100%;
+  width: 120px;
+}
+
+.desktopOnly {
+  display: none;
+  @include tablet-landscape {
+    display: flex;
+  }
+}
Index: astro_2.0/features/Bounties/helpers.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/helpers.ts b/astro_2.0/features/Bounties/helpers.tsx
rename from astro_2.0/features/Bounties/helpers.ts
rename to astro_2.0/features/Bounties/helpers.tsx
--- a/astro_2.0/features/Bounties/helpers.ts	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/astro_2.0/features/Bounties/helpers.tsx	(date 1643886656010)
@@ -1,5 +1,17 @@
 import { BountyContext } from 'types/bounties';
 import { differenceInMinutes } from 'date-fns';
+import { EXTERNAL_LINK_SEPARATOR } from 'constants/common';
+import {
+  SINGLE_BOUNTY_PAGE_URL,
+  SINGLE_PROPOSAL_PAGE_URL,
+} from 'constants/routing';
+import { VotingContent } from 'astro_2.0/features/Bounties/components/BountiesListView/components/VotingContent';
+import React from 'react';
+import { DAO } from 'types/dao';
+import { SectionItem } from 'astro_2.0/features/Bounties/components/BountiesListView/types';
+import { AmountContent } from 'astro_2.0/features/Bounties/components/BountiesListView/components/AmountContent';
+import { ProposalVariant } from 'types/proposal';
+import { Tokens } from 'context/CustomTokensContext';
 
 export const BOUNTIES_PAGE_SORT_OPTIONS = [
   {
@@ -63,11 +75,18 @@
 }
 
 export function prepareBountiesPageContent(
-  data: BountyContext[]
+  data: BountyContext[],
+  dao: DAO,
+  accountId: string,
+  completeHandler: (
+    id: number,
+    variant: ProposalVariant.ProposeDoneBounty
+  ) => void,
+  tokens: Tokens
 ): {
-  proposalPhase: BountyContext[];
-  bounties: BountyContext[];
-  completed: BountyContext[];
+  proposalPhase: SectionItem[];
+  bounties: SectionItem[];
+  completed: SectionItem[];
 } {
   const result = data.reduce<{
     proposalPhase: BountyContext[];
@@ -102,7 +121,91 @@
     }
   );
 
-  return result;
+  return {
+    proposalPhase: result.proposalPhase.map(item => {
+      const [description] = item.proposal.description.split(
+        EXTERNAL_LINK_SEPARATOR
+      );
+
+      return {
+        id: item.id,
+        title: description,
+        proposer: item.proposal.proposer,
+        proposalId: item.proposal.id,
+        link: {
+          pathname: SINGLE_PROPOSAL_PAGE_URL,
+          query: {
+            dao: dao.id,
+            proposal: item.proposal.id,
+          },
+        },
+        content: (
+          <VotingContent
+            proposal={item.proposal}
+            accountId={accountId}
+            dao={dao}
+          />
+        ),
+      };
+    }),
+    bounties: result.bounties.map(item => {
+      const [description] = item.bounty.description.split(
+        EXTERNAL_LINK_SEPARATOR
+      );
+
+      return {
+        id: item.id,
+        title: description,
+        proposer: item.proposal.proposer,
+        proposalId: item.proposal.id,
+        bounty: item.bounty,
+        link: {
+          pathname: SINGLE_BOUNTY_PAGE_URL,
+          query: {
+            dao: dao.id,
+            bounty: item.id,
+          },
+        },
+        completeHandler,
+        content: (
+          <AmountContent
+            tokens={tokens}
+            amount={item.bounty.amount}
+            token={item.bounty.token}
+            commentsCount={item.commentsCount}
+          />
+        ),
+      };
+    }),
+    completed: result.completed.map(item => {
+      const [description] = item.bounty.description.split(
+        EXTERNAL_LINK_SEPARATOR
+      );
+
+      return {
+        id: item.id,
+        title: description,
+        proposer: item.proposal.proposer,
+        proposalId: item.proposal.id,
+        bounty: item.bounty,
+        link: {
+          pathname: SINGLE_BOUNTY_PAGE_URL,
+          query: {
+            dao: dao.id,
+            bounty: item.id,
+          },
+        },
+        content: (
+          <AmountContent
+            tokens={tokens}
+            amount={item.bounty.amount}
+            token={item.bounty.token}
+            commentsCount={item.commentsCount}
+          />
+        ),
+      };
+    }),
+  };
 }
 
 export function getClaimProgress(
Index: components/Tabs/Tabs.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/components/Tabs/Tabs.tsx b/components/Tabs/Tabs.tsx
--- a/components/Tabs/Tabs.tsx	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/components/Tabs/Tabs.tsx	(date 1643885676897)
@@ -115,7 +115,7 @@
                 return (
                   <Tab
                     key={item.id}
-                    selectedClassName={styles.active}
+                    selectedClassName={cn(styles.active, item.activeClassName)}
                     className={cn(styles.tab, item.className)}
                     data-name={item.label}
                   >
Index: public/locales/en/common.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/public/locales/en/common.json b/public/locales/en/common.json
--- a/public/locales/en/common.json	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/public/locales/en/common.json	(date 1643882358801)
@@ -46,7 +46,10 @@
   "youFollowed": "You Followed",
   "confirmUnsubscribe": "Are you sure you want to unsubscribe from",
   "bountiesPage": {
-    "successClaimBountyNotification": "The blockchain transactions might take some time to perform, please visit the page in few seconds"
+    "successClaimBountyNotification": "The blockchain transactions might take some time to perform, please visit the page in few seconds",
+    "comingSoon": "Coming Soon",
+    "bounty": "Bounty",
+    "completed": "Completed"
   },
   "common": {
     "clickToUploadImage": "Click here to upload image",
Index: astro_2.0/features/Bounties/components/BountiesListView/components/CollapsableSection/CollpasableSection.module.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/components/BountiesListView/components/CollapsableSection/CollpasableSection.module.scss b/astro_2.0/features/Bounties/components/BountiesListView/components/CollapsableSection/CollpasableSection.module.scss
--- a/astro_2.0/features/Bounties/components/BountiesListView/components/CollapsableSection/CollpasableSection.module.scss	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/astro_2.0/features/Bounties/components/BountiesListView/components/CollapsableSection/CollpasableSection.module.scss	(date 1643905027879)
@@ -7,34 +7,61 @@
 }
 
 .header {
+  border-left: 12px solid var(--color-neutral-40);
+  border-radius: 8px 0 0;
   color: var(--color-neutral-60);
+  z-index: 0;
+
+  &.inProgress {
+    border-color: var(--color-brand-pink);
+  }
+
+  &.completed {
+    border-color: var(--color-success-20);
+  }
 }
 
 .header,
 .row {
   display: grid;
-  grid-column-gap: 12px;
-  grid-template-columns: 1fr minmax(48px, 120px) 1fr 1fr;
+  grid-column-gap: 4px;
+  grid-template-areas:
+    'title title title link'
+    'proposer proposer proposer proposer'
+    'toggle content content content';
+  grid-template-columns: 2fr 2fr 2fr 1fr;
+
+  @include tablet-landscape {
+    grid-column-gap: 12px;
+    grid-template-areas: unset;
+    grid-template-columns: 1fr 48px 80px 1fr 1fr;
+
+    &:last-of-type {
+      margin-bottom: 0;
+    }
+  }
 }
 
 .title,
 .link,
 .proposer,
 .content {
+  align-items: center;
+  display: flex;
   font-size: 14px;
   padding: 4px 8px;
 }
 
 .control {
-  left: -24px;
+  left: -32px;
   position: absolute;
-  top: 2px;
+  top: 4px;
 }
 
 .title {
   @extend %subtitle4;
   background-color: var(--color-neutral-40);
-  border-radius: 8px 8px 0 0;
+  border-radius: 0 8px 0 0;
   color: var(--color-brand-black);
 
   position: relative;
@@ -70,42 +97,84 @@
 }
 
 .row {
+  border-left: 12px solid var(--color-neutral-40);
   border-radius: 4px;
   box-shadow: 3px 2px 24px var(--color-black-opacity-1);
-  height: 56px;
   margin-bottom: 12px;
   overflow: hidden;
+  padding: 4px;
+
+  @include tablet-landscape {
+    height: 56px;
+  }
 
   &.firstRow {
     border-radius: 0 4px 4px;
   }
+
+  &.inProgress {
+    border-color: var(--color-brand-pink);
+  }
+
+  &.completed {
+    border-color: var(--color-success-20);
+  }
 }
 
 .rowTitle,
 .rowProposer,
 .rowLink,
+.toggle,
 .rowContent {
   align-items: center;
   display: flex;
   overflow: hidden;
-  padding: 16px 8px;
+  padding: 8px;
+
+  @include tablet-landscape {
+    padding: 16px 8px;
+  }
 }
 
 .rowTitle {
   @extend %title4;
-  border-left: 12px solid var(--color-neutral-40);
+  //border-left: 12px solid var(--color-neutral-40);
 
-  &.inProgress {
-    border-color: var(--color-brand-pink);
-  }
+  grid-area: title;
 
-  &.completed {
-    border-color: var(--color-success-20);
+  @include tablet-landscape {
+    grid-area: unset;
   }
 }
 
 .rowProposer {
   @extend %body2;
+
+  grid-area: proposer;
+
+  @include tablet-landscape {
+    grid-area: unset;
+  }
+}
+
+.rowLink {
+  grid-area: link;
+  justify-content: flex-end;
+
+  @include tablet-landscape {
+    grid-area: unset;
+    justify-content: flex-start;
+  }
+}
+
+.rowContent {
+  grid-area: content;
+  justify-content: flex-end;
+
+  @include tablet-landscape {
+    grid-area: unset;
+    justify-content: flex-start;
+  }
 }
 
 .singleLine {
@@ -151,6 +220,14 @@
   text-transform: capitalize;
 }
 
+.toggle {
+  grid-area: toggle;
+
+  @include tablet-landscape {
+    grid-area: unset;
+  }
+}
+
 .toggleClaims {
   align-items: center;
   border: 1px solid var(--color-neutral-60);
@@ -190,9 +267,13 @@
   background: var(--color-white);
   border-radius: 8px;
   box-shadow: 3px 2px 24px var(--color-black-opacity-1);
-  margin: 4px 24px;
-  padding: 8px 0;
+  margin: 4px 24px 12px;
+  padding: 0;
   position: relative;
+
+  @include tablet-landscape {
+    padding: 8px 0;
+  }
 }
 
 .claimsContainer::before {
Index: astro_2.0/features/Bounties/components/ViewToggle/ViewToggle.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/components/ViewToggle/ViewToggle.tsx b/astro_2.0/features/Bounties/components/ViewToggle/ViewToggle.tsx
--- a/astro_2.0/features/Bounties/components/ViewToggle/ViewToggle.tsx	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/astro_2.0/features/Bounties/components/ViewToggle/ViewToggle.tsx	(date 1643878644929)
@@ -10,11 +10,16 @@
 interface ViewToggleProps {
   selected: ViewToggleOption;
   onSelect: (val: ViewToggleOption) => void;
+  className?: string;
 }
 
-export const ViewToggle: FC<ViewToggleProps> = ({ selected, onSelect }) => {
+export const ViewToggle: FC<ViewToggleProps> = ({
+  selected,
+  onSelect,
+  className,
+}) => {
   return (
-    <div className={styles.root}>
+    <div className={cn(styles.root, className)}>
       <Button
         size="small"
         variant="tertiary"
Index: astro_2.0/features/Bounties/components/BountiesListView/components/CollapsableSection/CollapsableSection.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/components/BountiesListView/components/CollapsableSection/CollapsableSection.tsx b/astro_2.0/features/Bounties/components/BountiesListView/components/CollapsableSection/CollapsableSection.tsx
--- a/astro_2.0/features/Bounties/components/BountiesListView/components/CollapsableSection/CollapsableSection.tsx	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/astro_2.0/features/Bounties/components/BountiesListView/components/CollapsableSection/CollapsableSection.tsx	(date 1643891121231)
@@ -32,7 +32,13 @@
 
   return (
     <div className={styles.root}>
-      <div className={styles.header}>
+      <div
+        className={cn(styles.header, {
+          [styles.pending]: status === 'Pending',
+          [styles.inProgress]: status === 'InProgress',
+          [styles.completed]: status === 'Completed',
+        })}
+      >
         <div
           className={cn(styles.title, {
             [styles.pending]: status === 'Pending',
@@ -54,6 +60,7 @@
           {title}
         </div>
         <div className={styles.link} />
+        <div className={styles.toggle} />
         <div className={styles.proposer}>Proposer</div>
         <div className={styles.content}>{contentTitle}</div>
       </div>
Index: astro_2.0/features/Bounties/components/BountiesListView/BountiesListView.module.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/components/BountiesListView/BountiesListView.module.scss b/astro_2.0/features/Bounties/components/BountiesListView/BountiesListView.module.scss
--- a/astro_2.0/features/Bounties/components/BountiesListView/BountiesListView.module.scss	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/astro_2.0/features/Bounties/components/BountiesListView/BountiesListView.module.scss	(date 1643904915300)
@@ -1,3 +1,5 @@
+@import 'styles/mixins';
+
 .root {
   display: flex;
   flex-wrap: wrap;
@@ -10,6 +12,24 @@
 }
 
 .content {
-  margin-top: 24px;
+  margin-top: 0;
   width: 100%;
+
+  @include tablet-landscape {
+    margin-top: 24px;
+  }
+}
+
+.regular {
+  display: none;
+  @include tablet-landscape {
+    display: block;
+  }
+}
+
+.mobile {
+  display: block;
+  @include tablet-landscape {
+    display: none;
+  }
 }
Index: astro_2.0/features/Bounties/components/BountiesListView/BountiesListView.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/components/BountiesListView/BountiesListView.tsx b/astro_2.0/features/Bounties/components/BountiesListView/BountiesListView.tsx
--- a/astro_2.0/features/Bounties/components/BountiesListView/BountiesListView.tsx	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/astro_2.0/features/Bounties/components/BountiesListView/BountiesListView.tsx	(date 1643904915333)
@@ -1,17 +1,10 @@
 import React, { FC } from 'react';
-
+import cn from 'classnames';
 import { BountyContext } from 'types/bounties';
 
-import { EXTERNAL_LINK_SEPARATOR } from 'constants/common';
-import {
-  SINGLE_BOUNTY_PAGE_URL,
-  SINGLE_PROPOSAL_PAGE_URL,
-} from 'constants/routing';
-
 import { CollapsableSection } from 'astro_2.0/features/Bounties/components/BountiesListView/components/CollapsableSection';
 import { prepareBountiesPageContent } from 'astro_2.0/features/Bounties/helpers';
-import { VotingContent } from 'astro_2.0/features/Bounties/components/BountiesListView/components/VotingContent';
-import { AmountContent } from 'astro_2.0/features/Bounties/components/BountiesListView/components/AmountContent';
+import { MobileListView } from 'astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView';
 
 import { Tokens } from 'astro_2.0/features/CustomTokens/CustomTokensContext';
 
@@ -39,44 +32,23 @@
   tokens,
 }) => {
   const { proposalPhase, bounties, completed } = prepareBountiesPageContent(
-    bountiesContext
+    bountiesContext,
+    dao,
+    accountId,
+    completeHandler,
+    tokens
   );
 
   return (
     <div className={styles.root}>
-      <div className={styles.content}>
+      <div className={cn(styles.content, styles.regular)}>
         <CollapsableSection
           status="Pending"
           accountId={accountId}
           title={`Coming Soon (Proposal Phase) (${proposalPhase.length})`}
           contentTitle="Voting"
           dao={dao}
-          data={proposalPhase.map(item => {
-            const [description] = item.proposal.description.split(
-              EXTERNAL_LINK_SEPARATOR
-            );
-
-            return {
-              id: item.id,
-              title: description,
-              proposer: item.proposal.proposer,
-              proposalId: item.proposal.id,
-              link: {
-                pathname: SINGLE_PROPOSAL_PAGE_URL,
-                query: {
-                  dao: dao.id,
-                  proposal: item.proposal.id,
-                },
-              },
-              content: (
-                <VotingContent
-                  proposal={item.proposal}
-                  accountId={accountId}
-                  dao={dao}
-                />
-              ),
-            };
-          })}
+          data={proposalPhase}
         />
         <CollapsableSection
           status="InProgress"
@@ -84,35 +56,7 @@
           title={`Bounties (${bounties.length})`}
           contentTitle="Amount"
           dao={dao}
-          data={bounties.map(item => {
-            const [description] = item.bounty.description.split(
-              EXTERNAL_LINK_SEPARATOR
-            );
-
-            return {
-              id: item.id,
-              title: description,
-              proposer: item.proposal.proposer,
-              proposalId: item.proposal.id,
-              bounty: item.bounty,
-              link: {
-                pathname: SINGLE_BOUNTY_PAGE_URL,
-                query: {
-                  dao: dao.id,
-                  bounty: item.id,
-                },
-              },
-              completeHandler,
-              content: (
-                <AmountContent
-                  tokens={tokens}
-                  amount={item.bounty.amount}
-                  token={item.bounty.token}
-                  commentsCount={item.commentsCount}
-                />
-              ),
-            };
-          })}
+          data={bounties}
         />
         <CollapsableSection
           status="Completed"
@@ -120,34 +64,16 @@
           title={`Completed (${completed.length})`}
           contentTitle="Amount"
           dao={dao}
-          data={completed.map(item => {
-            const [description] = item.bounty.description.split(
-              EXTERNAL_LINK_SEPARATOR
-            );
-
-            return {
-              id: item.id,
-              title: description,
-              proposer: item.proposal.proposer,
-              proposalId: item.proposal.id,
-              bounty: item.bounty,
-              link: {
-                pathname: SINGLE_BOUNTY_PAGE_URL,
-                query: {
-                  dao: dao.id,
-                  bounty: item.id,
-                },
-              },
-              content: (
-                <AmountContent
-                  tokens={tokens}
-                  amount={item.bounty.amount}
-                  token={item.bounty.token}
-                  commentsCount={item.commentsCount}
-                />
-              ),
-            };
-          })}
+          data={completed}
+        />
+      </div>
+      <div className={cn(styles.content, styles.mobile)}>
+        <MobileListView
+          proposals={proposalPhase}
+          bounties={bounties}
+          completed={completed}
+          dao={dao}
+          accountId={accountId}
         />
       </div>
     </div>
Index: astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/MobileListView.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/MobileListView.tsx b/astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/MobileListView.tsx
new file mode 100644
--- /dev/null	(date 1643904896482)
+++ b/astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/MobileListView.tsx	(date 1643904896482)
@@ -0,0 +1,88 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'next-i18next';
+
+import { Tabs } from 'components/Tabs';
+import { SectionItem } from 'astro_2.0/features/Bounties/components/BountiesListView/types';
+import { SectionRow } from 'astro_2.0/features/Bounties/components/BountiesListView/components/SectionRow';
+import { NoResultsView } from 'astro_2.0/components/NoResultsView';
+
+import { DAO } from 'types/dao';
+
+import styles from './MobileListView.module.scss';
+
+interface MobileListViewProps {
+  proposals: SectionItem[];
+  bounties: SectionItem[];
+  completed: SectionItem[];
+  accountId: string;
+  dao: DAO;
+}
+
+export const MobileListView: FC<MobileListViewProps> = ({
+  proposals,
+  bounties,
+  completed,
+  accountId,
+  dao,
+}) => {
+  const { t } = useTranslation();
+
+  function getTabContent(
+    items: SectionItem[],
+    status: 'Pending' | 'InProgress' | 'Completed'
+  ) {
+    return (
+      <div className={styles.tabContent}>
+        {!items.length && <NoResultsView title="No results found" />}
+        {items.map(item => {
+          return (
+            <SectionRow
+              id={item.id}
+              key={item.id}
+              accountId={accountId}
+              dao={dao}
+              isFirstRow={false}
+              item={item}
+              status={status}
+            />
+          );
+        })}
+      </div>
+    );
+  }
+
+  const TABS = [
+    {
+      id: 0,
+      label: t(`bountiesPage.comingSoon`),
+      content: getTabContent(proposals, 'Pending'),
+      className: styles.tabsListItem,
+      activeClassName: styles.activeTabsListItem,
+    },
+    {
+      id: 1,
+      label: t(`bountiesPage.bounty`),
+      content: getTabContent(bounties, 'InProgress'),
+      className: styles.tabsListItem,
+      activeClassName: styles.activeTabsListItem,
+    },
+    {
+      id: 2,
+      label: t(`bountiesPage.completed`),
+      content: getTabContent(completed, 'Completed'),
+      className: styles.tabsListItem,
+      activeClassName: styles.activeTabsListItem,
+    },
+  ];
+
+  return (
+    <div className={styles.root}>
+      <Tabs
+        skipShallow
+        tabs={TABS}
+        tabsWrapperClassName={styles.tabsRoot}
+        tabsListRootClassName={styles.tabsListRoot}
+      />
+    </div>
+  );
+};
Index: astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/index.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/index.ts b/astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/index.ts
new file mode 100644
--- /dev/null	(date 1643881252477)
+++ b/astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/index.ts	(date 1643881252477)
@@ -0,0 +1,1 @@
+export * from './MobileListView';
Index: astro_2.0/features/pages/nestedDaoPagesContent/BountiesPageContent/BountiesPageContent.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/pages/nestedDaoPagesContent/BountiesPageContent/BountiesPageContent.tsx b/astro_2.0/features/pages/nestedDaoPagesContent/BountiesPageContent/BountiesPageContent.tsx
--- a/astro_2.0/features/pages/nestedDaoPagesContent/BountiesPageContent/BountiesPageContent.tsx	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/astro_2.0/features/pages/nestedDaoPagesContent/BountiesPageContent/BountiesPageContent.tsx	(date 1643904929888)
@@ -1,5 +1,6 @@
 import React, { useState, VFC } from 'react';
 import useQuery from 'hooks/useQuery';
+import cn from 'classnames';
 
 import { useAuthContext } from 'context/AuthContext';
 import { DaoContext } from 'types/context';
@@ -73,9 +74,10 @@
         </Button>
 
         <div className={styles.filters}>
-          <div className={styles.filter}>
+          <div className={cn(styles.filter, styles.desktopOnly)}>
             <span className={styles.filterLabel}>Sorting by:</span>
             <Dropdown
+              menuClassName={styles.filterMenu}
               value={query.bountySort ?? BOUNTIES_PAGE_SORT_OPTIONS[0].value}
               onChange={val => updateQuery('bountySort', val)}
               options={BOUNTIES_PAGE_SORT_OPTIONS}
@@ -84,6 +86,7 @@
           <div className={styles.filter}>
             <span className={styles.filterLabel}>Filter:</span>
             <Dropdown
+              menuClassName={styles.filterMenu}
               value={
                 query.bountyFilter ?? BOUNTIES_PAGE_FILTER_OPTIONS[0].value
               }
@@ -93,7 +96,11 @@
           </div>
         </div>
 
-        <ViewToggle onSelect={setActiveView} selected={activeView} />
+        <ViewToggle
+          onSelect={setActiveView}
+          selected={activeView}
+          className={styles.desktopOnly}
+        />
       </div>
 
       {activeView === 'list' && (
Index: astro_2.0/features/Bounties/components/BountiesListView/components/ClaimRow/ClaimRow.module.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/components/BountiesListView/components/ClaimRow/ClaimRow.module.scss b/astro_2.0/features/Bounties/components/BountiesListView/components/ClaimRow/ClaimRow.module.scss
--- a/astro_2.0/features/Bounties/components/BountiesListView/components/ClaimRow/ClaimRow.module.scss	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/astro_2.0/features/Bounties/components/BountiesListView/components/ClaimRow/ClaimRow.module.scss	(date 1643904941115)
@@ -7,10 +7,6 @@
   justify-content: flex-start;
   position: relative;
   width: 100%;
-
-  &:not(:last-of-type) {
-    margin-bottom: 8px;
-  }
 }
 
 .root::before {
@@ -29,8 +25,12 @@
 .legend {
   border-radius: 50%;
   height: 8px;
-  margin: 24px 12px;
+  margin: 14px 12px;
   width: 8px;
+
+  @include tablet-landscape {
+    margin: 24px 12px;
+  }
 }
 
 .root.inProgress {
@@ -94,14 +94,26 @@
 }
 
 .person {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
   overflow: hidden;
   padding: 8px 12px 8px 0;
-  width: 200px;
+  width: 130px;
+
+  @include tablet-landscape {
+    width: 200px;
+  }
 
   .label {
     @extend %caption1;
     color: var(--color-neutral-60);
+    display: none;
     margin-bottom: 4px;
+
+    @include tablet-landscape {
+      display: block;
+    }
   }
 
   .value {
@@ -112,12 +124,16 @@
 }
 
 .details {
-  display: flex;
+  display: none;
   flex-direction: column;
   justify-content: center;
   padding: 8px 12px;
   width: 194px;
 
+  @include tablet-landscape {
+    display: flex;
+  }
+
   .icon {
     width: 16px;
   }
@@ -161,7 +177,9 @@
   color: var(--color-brand-black);
   display: flex;
 
-  margin-left: 48px;
+  @include tablet-landscape {
+    margin-left: 48px;
+  }
 
   .proposalLink {
     margin-left: 8px;
Index: astro_2.0/features/Bounties/components/BountiesListView/components/SectionRow/SectionRow.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/components/BountiesListView/components/SectionRow/SectionRow.tsx b/astro_2.0/features/Bounties/components/BountiesListView/components/SectionRow/SectionRow.tsx
--- a/astro_2.0/features/Bounties/components/BountiesListView/components/SectionRow/SectionRow.tsx	(revision ed022e92e31f30063efa60d897e62d8722458768)
+++ b/astro_2.0/features/Bounties/components/BountiesListView/components/SectionRow/SectionRow.tsx	(date 1643889284330)
@@ -42,15 +42,12 @@
         exit={{ opacity: 0 }}
         className={cn(styles.row, {
           [styles.firstRow]: isFirstRow,
-        })}
-      >
-        <div
-          className={cn(styles.rowTitle, {
-            [styles.pending]: status === 'Pending',
-            [styles.inProgress]: status === 'InProgress',
-            [styles.completed]: status === 'Completed',
-          })}
-        >
+          [styles.pending]: status === 'Pending',
+          [styles.inProgress]: status === 'InProgress',
+          [styles.completed]: status === 'Completed',
+        })}
+      >
+        <div className={cn(styles.rowTitle)}>
           <Link href={item.link}>
             <a className={styles.singleLine}>
               <div className={styles.singleLine}>{item.title}</div>
@@ -71,7 +68,9 @@
               <Icon name="buttonExternal" className={styles.icon} />
             </a>
           </Link>
+        </div>
 
+        <div className={styles.toggle}>
           {item.bounty && (
             <Button
               size="small"
@@ -95,6 +94,7 @@
             </Button>
           )}
         </div>
+
         <div className={styles.rowProposer}>
           <div className={styles.singleLine}>{item.proposer}</div>
         </div>
Index: astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/MobileListView.module.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/MobileListView.module.scss b/astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/MobileListView.module.scss
new file mode 100644
--- /dev/null	(date 1643904551402)
+++ b/astro_2.0/features/Bounties/components/BountiesListView/components/MobileListView/MobileListView.module.scss	(date 1643904551402)
@@ -0,0 +1,18 @@
+@import 'styles/typography';
+
+.tabs {
+  justify-content: flex-start;
+}
+
+.tabsListItem {
+  color: var(--color-neutral-50);
+  font-size: 16px;
+}
+
+.activeTabsListItem {
+  color: var(--color-neutral-80);
+}
+
+.tabContent {
+  margin-bottom: 80px;
+}
